buildSrc/src/main/java/Config.kt

```diff
@@ -0,0 +1,8 @@
+object Version {
+    const val compileSdkVersion = 28
+    const val minSdkVersion14 = 14
+    const val targetSdkVersion = 28
+    const val buildToolsVersion = "28.0.1"
+    const val supportVersion = "27.1.1"
```

Comment:
The support/arch libs are going to have separate release cycles since androidx, so don't think it's worth pulling out a common version for them. Also, any particular reason for these versions? Want to keep with the lowest ones that work.